packages/bpk-docs/src/pages/AlignmentPage/AlignmentPage.js

```diff
@@ -163,6 +165,41 @@ const AlignedSpan = withAlignment(
       </BpkText>,
     ],
   },
+  {
+    id: 'smallWithIcon',
+    title: 'Text with textStyle "sm" alignment with an icon',
+    blurb: [
+      <Paragraph>
+        Similarly, to use a small text, line height and the BpkText wrapper need
+        to use the small size:
```

Comment:
not all the cases are covered in examples so important to inform that the elements to play with are the size in the BpkText and the line height